integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -405,6 +405,42 @@ tieredstore:
     high: 0.95
     low: 0.7
 
+## Proxy ##
+proxy:
+  enabled: false # Enable this to enable the proxy for REST API
+  env:
+  # Extra environment variables for the Proxy pod
+  # Example:
+  # JAVA_HOME: /opt/java
+  args:
+    - proxy
+  # Properties for the proxy component
+  properties:
+  resources:
+    requests:
+      cpu: "0.5"
+      memory: "1"
```

Comment:
Is memory "1" intended?

Reply:
Thanks for catching this, I will fix it.